sdk/eventhub/azure-eventhub/stress/azure_eventhub_consumer_stress_async.py

```diff
@@ -0,0 +1,192 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+import time
+import argparse
+import asyncio
+import os
+import logging
+from collections import defaultdict
+
+from azure.identity.aio import ClientSecretCredential
+from azure.eventhub.aio import EventHubConsumerClient
+from azure.eventhub import EventHubSharedKeyCredential
+from azure.eventhub.extensions.checkpointstoreblobaio import BlobCheckpointStore
+from azure.eventhub import TransportType
+
+from logger import get_logger
+from process_monitor import ProcessMonitor
+
+
+parser = argparse.ArgumentParser()
+parser.add_argument("--link_credit", default=3000, type=int)
+parser.add_argument("--output_interval", type=float, default=1000)
+parser.add_argument("--duration", help="Duration in seconds of the test", type=int, default=30)
+parser.add_argument("--consumer_group", help="Consumer group name", default="$default")
+parser.add_argument("--auth_timeout", help="Authorization Timeout", type=float, default=60)
+parser.add_argument("--offset", help="Starting offset", default="-1")
+parser.add_argument("--partitions", help="Number of partitions. 0 means to get partitions from eventhubs", type=int, default=0)
+parser.add_argument("--recv_partition_id", help="Receive from a specific partition if this is set", type=int)
+parser.add_argument("--track_last_enqueued_event_properties", action="store_true")
+parser.add_argument("--load_balancing_interval", help="time duration in seconds between two load balance", type=float, default=10)
+parser.add_argument("--conn_str", help="EventHub connection string",
+                    default=os.environ.get('EVENT_HUB_PERF_32_CONN_STR'))
+parser.add_argument("--eventhub", help="Name of EventHub")
+parser.add_argument("--address", help="Address URI to the EventHub entity")
+parser.add_argument("--sas_policy", help="Name of the shared access policy to authenticate with")
+parser.add_argument("--sas_key", help="Shared access key")
+parser.add_argument(
+    "--transport_type",
+    help="Transport type, 0 means AMQP, 1 means AMQP over WebSocket",
+    type=int,
+    default=0
+)
+parser.add_argument("--parallel_recv_cnt", help="Number of parallelling receiving", type=int)
```

Comment:
What is this?

Reply:
This is for creating multiple clients to receive simultaneously